src/database/DataTypes/Transaction.js

```diff
@@ -449,6 +457,13 @@ export class Transaction extends Realm.Object {
     }
     if (!this.isConfirmed) this.confirm(database);
 
+    // Finding the totalPrice propogates through `TransactionItem` records down to the batch
+    // level, deriving the full cost of the Transaction. Cash transactions and credits are
+    // created finalised, having their total already set.
+    if (!this.isCashTransaction || !this.isCredit) {
```

Comment:
Should this be `!(this.isCashTransaction || this.isCredit)`

Reply:
![embares](https://user-images.githubusercontent.com/35858975/73804263-0a7bd200-4828-11ea-9496-7f9fc191c763.gif)

https://github.com/openmsupply/mobile/pull/2136/commits/e7870f102c8abf3ecfd77e57f4a961c1c1f633b4